Services/FileDelivery/classes/Delivery.php

```diff
@@ -639,7 +639,7 @@ public static function returnASCIIFileName($original_filename) {
 
 		$ascii_filename = htmlentities($original_filename, ENT_NOQUOTES, 'UTF-8');
 		$ascii_filename = preg_replace('/\&(.)[^;]*;/', '\\1', $ascii_filename);
-		$ascii_filename = preg_replace('/[\x7f-\xff]/', '_', $ascii_filename);
+		$ascii_filename = preg_replace('/[\x00-\x1F\x7F]/u', '_', $ascii_filename);
```

Comment:
Hi @ralfzeit, thanks for your contribution. I'm sure the changes will fix your specific issue of characters which are replaced. But I think this change will break the replacement for other characters as well. Delivered files should have proper ASCII-filenames, otherwise this can lead to problems. But if you need the original filename with non ASCI characters, you can disable the conversion bu putting some lines in your client.ini.php, see https://github.com/ILIAS-eLearning/ILIAS/blob/release_5-3/docs/configuration/settings/ini-file.md#clientiniphp : 
```ini
[file_access] 
disable_ascii = "0"
```
could you please check if this solves your problem as well?

Reply:
Hi @chfsx !

I've setted up this setting, but it didn't solve my problem.